lib/elixir/lib/kernel.ex

```diff
@@ -4737,8 +4778,8 @@ defmodule Kernel do
 
         _otherwise ->
           raise ArgumentError,
-                "invalid arguments for use, " <>
-                  "expected a compile time atom or alias, got: #{Macro.to_string(module)}"
+                <<"invalid arguments for use, ",
```

Comment:
Same here: we don't need to change this anymore?

Reply:
The ones on other files now work, but those still don't:
```
==> elixir (compile)
Compiled src/elixir_parser.yrl
Compiled src/elixir_code_server.erl
Compiled src/elixir_lexical.erl
Compiled src/elixir_bitstring.erl
Compiled src/elixir_erl_for.erl
Compiled src/elixir_interpolation.erl
Compiled src/elixir_module.erl
Compiled src/elixir_quote.erl
Compiled src/elixir_sup.erl
Compiled src/elixir_bootstrap.erl
Compiled src/elixir_rewrite.erl
Compiled src/elixir_locals.erl
Compiled src/elixir_erl.erl
Compiled src/elixir_erl_compiler.erl
Compiled src/elixir_overridable.erl
Compiled src/elixir.erl
Compiled src/elixir_erl_clauses.erl
Compiled src/elixir_fn.erl
Compiled src/elixir_clauses.erl
Compiled src/elixir_map.erl
Compiled src/elixir_def.erl
Compiled src/elixir_env.erl
Compiled src/elixir_compiler.erl
Compiled src/elixir_erl_var.erl
Compiled src/elixir_utils.erl
Compiled src/elixir_import.erl
Compiled src/elixir_dispatch.erl
Compiled src/elixir_expand.erl
Compiled src/elixir_errors.erl
Compiled src/elixir_aliases.erl
Compiled src/elixir_config.erl
Compiled src/elixir_tokenizer.erl
Compiled src/elixir_erl_try.erl
Compiled src/elixir_erl_pass.erl
Compiled src/elixir_parser.erl
==> bootstrap (compile)
error: undef
stacktrace: [{'Elixir.Kernel','bootstrapped?',1,[]},
             {erl_eval,local_func,6,[{file,"erl_eval.erl"},{line,557}]},
             {erl_eval,expr_list,6,[{file,"erl_eval.erl"},{line,878}]},
             {erl_eval,expr,5,[{file,"erl_eval.erl"},{line,236}]},
             {erl_eval,expr,5,[{file,"erl_eval.erl"},{line,269}]},
             {erl_eval,expr,5,[{file,"erl_eval.erl"},{line,438}]},
             {erl_eval,exprs,5,[{file,"erl_eval.erl"},{line,122}]},
             {'Elixir.Kernel','<>',2,[{file,"expanding macro"}]}]
make: *** [lib/elixir/ebin/Elixir.Kernel.beam] Error 1
```
Only if we change the place were this is defined